### DISCUSSION TITLE
Keeshare on M1 Mac only allowing unsigned sharing

### DISCUSSION THREAD

**Discussion by axelson** [keepassxreboot/keepassxc#7285] [Q&A] [answered]
I have an m1 MacBook air but I'm not able to get Keeshare fully working on it. It's running 2.6.6 but the keeshare extension/plugin says "only unsigned sharing" (in the debug info). I wasn't able to find a matching issue in the repository. Is there some sort of configuration issue?

**Comment by phoerious** [comment, marked as answer]
That's an issue with Quazip not being available in the correct version in Homebrew. In general, you don't need signed sharing. It adds no additional protection and we will remove that feature in a future version anyway.

**Reply by axelson** [reply]
Ah, okay. Thanks for the info!
